### PR TITLE
Issue #164/165/168: Update UG/DG for sort, find and view 

### LINKED ISSUE #164: Update `find` command in UG/DG
Updates to be made to (but not limited to) user stories, use cases, implementation, features sections.

### LINKED ISSUE #165: Update `sort` command in UG/DG
Updates to be made to (but not limited to) user stories, use cases, implementation, features sections.

### LINKED ISSUE #168: Update `view` command in UG/DG
Updates to be made to (but not limited to) user stories, use cases, implementation, features sections.

Tag teaming here, let's update each other on progress and see what we can get done! :)

### LINKED ISSUE #217: Change UI of `Candidate List` to follow `Interview List`

### PR DESCRIPTION
**Pull request by charmainehly**
Still a work in progress. 

**Currently PR includes:**
- (UG) additions of the above commands to the features section of UG
- (UG) midpoint update of gui template
- (GUI) minor edits to gui to standardise the colour scheme and style
- (Functionality) shortening of attribute fields used in the sort and find command to standardise with other commands
- (Functionality) fixing bug for find command for availability (to take out the `availability: ` string in the validity check)
- (EDIT) Shifted priority of `sort` command in user stories from `low` to `medium`
- (WIP) Implementation of `sort` and `view` commands
- (WIP) Refactoring of TAs to candidates

<br> **Following points up for discussion!!! (to delete, modify etc.!!)**
- (UG) reorganisation of command summary for user's ease of understanding
- (UG) addition of a section on how to navigate the display (i.e. our three panels)

<br> Closes #168 , Closes #164 , Closes #165 , Closes #217  - will open issues for finalising of DG and WIP sections!

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -4,9 +4,13 @@ title: User Guide
 ---
 
 TAlent Assistant™ is a **desktop, lightweight and centralized management system** catered to professors for managing
-the process of hiring undergraduate/graduate Teaching Assistants (TA). They will be able to access the candidates’ application
-data easily and review qualifications or availability for scheduling interviews. It is **optimized for use via a
-Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, this application will be able to help you manage all things under the hood of the TA initiative faster than traditional GUI applications.
+the interview scheduling process of candidates applying to be undergraduate/graduate Teaching Assistants (TA).
```
Comment (charmainehly) [inline docs/UserGuide.md:7]: Rationale of change - to improve phrasing and emphasise on core functionality/value proposition being the scheduling functionality. Qualifications of candidates is out of scope.

Adjusted phrasing to make subject of sentence clearer.

**Comment by charmainehly** [comment]
@domlimm @LeeEnEn good to go for merging once yall have reviewed! - i'll be opening new issues for outstanding tasks on the DG so as to update progress! The failing `codecov` is attributed to the edits to code lines, with no additional test cases added.

**Comment by domlimm** [comment]
> @domlimm @LeeEnEn good to go for merging once yall have reviewed! - i'll be opening new issues for outstanding tasks on the DG so as to update progress! The failing `codecov` is attributed to the edits to code lines, with no additional test cases added.

@charmainehly Noted! Was waiting for your go to review and merge after seeing you assigned us but stated that there were more things to add. I'll review right now! 👍🏻

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -375,16 +375,16 @@ The main logic for `find` command is still implemented within `FindCommand` (whi
 
 The `FindCommandParser` class has been extended to include a switch-case block to validate
 which field the user is trying to search across:
-1. `studentId`
+1. `studentId` (accepted user input abbreviated as `id`)
 2. `course`
```
Comment (domlimm) [inline docs/DeveloperGuide.md:379]: Would we need this?

```suggestion
2. `course` (accepted user input abbreviated as `c`)
```

File: docs/UserGuide.md (hunk 1 above)
Comment (domlimm) [reply docs/UserGuide.md:7]: Agreed!

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java [hunk 3]
```diff
@@ -53,26 +53,26 @@ public FindCommand parse(String args) throws ParseException {
         }
 
         switch (fieldString) {
```
Comment (domlimm) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java:55]: Love that you changed this to ensure it's consistent! 😄 👍🏻

File: src/main/java/seedu/address/model/candidate/predicate/CandidateContainsKeywordsPredicate.java [hunk 4]
```diff
@@ -29,18 +32,12 @@ public CandidateContainsKeywordsPredicate(List<String> keywords) {
      */
     @Override
     public boolean test(Candidate candidate) {
-        StringBuilder sb = new StringBuilder();
-        String[] separatedCandidate = candidate.toString().split("Availability: ");
-        int[] availArr = Arrays.stream(separatedCandidate[1].split(",")).mapToInt((Integer::parseInt)).toArray();
-
-        sb.append(separatedCandidate[0]).append(" Availability: ");
+        String availability = candidate.getAvailability().toString();
```
Comment (domlimm) [inline src/main/java/seedu/address/model/candidate/predicate/CandidateContainsKeywordsPredicate.java:35]: Nice refactor!

File: src/main/resources/view/DarkTheme.css [hunk 5]
```diff
@@ -493,8 +493,7 @@
 
 // ************* interview panel css *************
    .interview-panel-header {
-       -fx-background-color: derive(#353C51, 10%);
-   }
+    -fx-background-color: #3c3e3f;   }
```
Comment (domlimm) [inline src/main/resources/view/DarkTheme.css:496]: The bracket `}` at the end should probably be on the next line right? 🙈

**Review by domlimm — CHANGES_REQUESTED** [review]
@charmainehly  Thanks for refactoring `TA/s` to `candidate/s`! All LGTM! Good job! 💯 

Was thinking for the `docs/images/Ui.png`, we could add a more in depth image showing:
1. Some interviews scheduled
2. Focused on a candidate
3. Show some command typed in with the feedback filled

File: src/main/resources/view/DarkTheme.css (hunk 5 above)
Comment (charmainehly) [reply src/main/resources/view/DarkTheme.css:496]: will make the change for this!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (charmainehly) [reply docs/DeveloperGuide.md:379]: not relevant here! Have kept the user input for find command as `course`! will update if we deem it necessary to shorten `course` to `c` as well?

**Comment by charmainehly** [comment]
Thanks @domlimm have addressed the comments. Agreed with the change to Ui.png, but lets finalise it at the end? :)

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (domlimm) [reply docs/DeveloperGuide.md:379]: For `add` and `edit`, both are using `c` char

**Review by domlimm — APPROVED** [review]
Thanks for making the change! LGTM! 👍🏻

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (charmainehly) [reply docs/DeveloperGuide.md:379]: this section is for the `find` command! not the `edit` and `add`! Hmmm, unless your suggestion is to change `course` to `c`, `phone` to `p` and `email` to `e` as well for the `find` command?

File: docs/UserGuide.md (hunk 1 above)
Comment (LeeEnEn) [reply docs/UserGuide.md:7]: Yep, I agree too!

File: docs/UserGuide.md [hunk 6]
```diff
@@ -35,9 +39,11 @@ Command Line Interface (CLI)** while still having the benefits of a Graphical Us
 
     * **`delete 1`** : Deletes the first candidate in the system.
 
-    * **`find k/Jane f/name`** : Searches for all candidate with name containing “Jane/jane”.
+    * **`find k/Alex f/name`** : Searches for all candidate with name containing “jane" (case-insensitive).
```
Comment (LeeEnEn) [inline docs/UserGuide.md:42]: Perhaps the `"jane"` should be `alex` instead? 🤔

Comment (charmainehly) [reply docs/UserGuide.md:42]: oh yah cos loading from the sample data right! alright lemme make that change!! :-) thank you!!!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (domlimm) [reply docs/DeveloperGuide.md:379]: hey @charmainehly, yup i know it is for that. i'm okay either ways. i just thought of the consistency between all the commands that's all. tbh, if no t ime, let's let it go. 👍🏻 i'll let you decide boss!

**Pull request merged**